video_emotion/facial_recognition/t/test_facial_recognition.py

```diff
@@ -0,0 +1,38 @@
+import os
+import cv2
+
+from video_emotion.facial_recognition.facial_recognition import analyze_video, analyze_frame
+
+def test_facial_recognition_of_at_least_one_face():
+    dirname = os.path.dirname(__file__)
+    test_filename = os.path.join(
+        dirname, "test_facial_recognition/Fun_at_a_Fair.mp4")
+
+    output_frames = analyze_video(test_filename, (3000, 5000))
+
+    assert len(output_frames) != 0
+
+    frame_key = "79"
```

Comment:
Huh? Magic number?

Reply:
If you want to comply with what Casper is exclaiming, please add a unit to your comment. Like _analyze from 3 to 5 seconds..._